packages/snaps-simulator/src/features/builder/utils.test.ts

```diff
@@ -55,7 +55,7 @@ describe('isValidFieldChild', () => {
 
   it('returns false for invalid field children', () => {
     const child = Text({ children: 'foo' });
-    expect(isValidFieldChild(child)).toBe(false);
+    expect(isValidFieldChild(child)).toBe(true);
```

Comment:
This test seems flawed now 😄 I guess the only invalid field child now would be `Option`?

Reply:
Replaced with `<Option>`.